src/main/java/gift/service/MemberService.java

```diff
@@ -3,25 +3,25 @@
 import gift.model.Member;
 import gift.model.dto.LoginMemberDto;
 import gift.model.dto.MemberRequestDto;
-import gift.repository.MemberDao;
+import gift.repository.MemberRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 public class MemberService {
 
-    private final MemberDao memberDao;
+    private final MemberRepository memberRepository;
 
-    public MemberService(MemberDao memberDao) {
-        this.memberDao = memberDao;
+    public MemberService(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
     }
 
     public LoginMemberDto selectLoginMemberById(Long id) {
-        Member member = memberDao.selectMemberById(id);
+        Member member = memberRepository.findById(id).get();
         return new LoginMemberDto(member.getId(), member.getName(), member.getEmail(),
             member.getRole());
     }
```

Comment:
바로 Optional.get()을 호출하면 NPE가 발생할 수 있어요.  

```suggestion
        return memberRepository.findById(id)
                .map(v -> new LoginMemberDto(v.getId(), v.getName(), v.getEmail(), v.getRole()))
                .orElseThrow(() -> new IllegalArgumentException("..."));
```

이렇게도 표현해볼 수 있습니다.  

추가로 LoginMemberDto가 Member를 받아 생성해주는 생성자 혹은 정적 팩토리 메서드를 추가해줘도 좋겠네요 :)  
https://tecoble.techcourse.co.kr/post/2020-05-26-static-factory-method/